parser-typechecker/src/Unison/Codebase/Branch.hs

```diff
@@ -618,13 +700,13 @@ addTypeName r new md =
 deleteTermName :: Referent -> NameSegment -> Branch0 m -> Branch0 m
 deleteTermName r n b
   | Star3.memberD1 (r, n) (view terms b) =
-      over terms (Star3.deletePrimaryD1 (r, n)) b
+    over terms (Star3.deletePrimaryD1 (r, n)) b
```

Comment:
Pre-0.4.0.0 `ormolu` is back from the grave

Reply:
Sorry... I'm not actually sure how to fix this the ormolu issue.
```suggestion
      over terms (Star3.deletePrimaryD1 (r, n)) b
```